notifier/prometheus.go

```diff
@@ -63,7 +64,8 @@ func SendDingTalkNotification(httpClient *http.Client, webhookURL string, notifi
 	var robotResp models.DingTalkNotificationResponse
 	enc := json.NewDecoder(req.Body)
 	if err := enc.Decode(&robotResp); err != nil {
-		return nil, errors.Wrap(err, "error decoding response from DingTalk")
+		body, err = ioutil.ReadAll(req.Body)
```

Comment:
When using a decoder, you cannot retrieve full body if the json decoder failed in the middle.
A better way is just read the body into `[]byte`, then call `json.Unmarshal()` directly.

Reply:
Actually, I don't think it's a good idea to log response body, though